java/partitions/splitterv2/src/main/java/sleeper/splitterv2/core/split/FindPartitionSplitPoint.java

```diff
@@ -110,13 +110,12 @@ public interface SketchesLoader {
         Sketches load(String filename) throws IOException;
     }
 
-    public static SketchesLoader loadSketchesFromFile(Schema schema, Configuration conf) {
-        return (filename) -> new SketchesSerDeToS3(schema).loadFromHadoopFS(new Path(filename), conf);
+    public static SketchesLoader loadSketchesFromFile(Schema schema) {
+        return (filename) -> new S3SketchesStore(S3Client.create(), S3TransferManager.create()).loadFileSketches(filename, schema);
```

Comment:
It looks like the v1 version of this method is unused, so this can probably be deleted.